src/components/Progress/Progress.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+const calculateProgressPercent = (progress, childrenCount) => {
+  if (progress === 0 || childrenCount === 0) return 0;
+
+  const maxPossibleProgress = Math.min(progress, childrenCount);
+
+  if (maxPossibleProgress >= Math.max(childrenCount - 1, 0)) return 100;
+
+  return maxPossibleProgress / (childrenCount - 1) * 100;
+};
+
+const Progress = ({
+  assistiveText,
+  children,
+  flavor,
+  isVertical,
+  progress,
+  progressLabel,
+}) => {
+  const progressPercent = calculateProgressPercent(
+    progress,
+    React.Children.count(children)
+  );
+
+  return (
+    <div>
```

Comment:
Intentional wrapper div?

Reply:
Nope, leftover of the popup as SLDS implemented it. Can be removed for now. 👍